piksi_multi_cpp/src/sbp_callback_handler/sbp_callback_handler_relay/ros_enu_relays.cc

```diff
@@ -23,4 +23,62 @@ bool RosTransformEnuRelay::convertSbpMsgToRosMsg(
   return true;
 }
 
+bool RosPositionWithCovarianceEnuRelay::convertSbpMsgToRosMsg(
+    const msg_pos_ecef_cov_t& in, const uint8_t len,
+    piksi_rtk_msgs::PositionWithCovarianceStamped* out) {
+  ROS_ASSERT(out);
+
+  if (!convertEcefToEnu(in, &out->position.position)) return false;
```

Comment:
Style Question: if statement without brackets

Reply:
See above. Ok for short single line statements.